src/components/SelectBar.js

```diff
@@ -75,6 +82,20 @@ const SelectBar = ({ changeLanguage }) => {
           handleClick(e, 'py');
         }}
       />
+      <img
+        src={
+          lang === 'js'
+            ? `${process.env.PUBLIC_URL}/assets/js-active.png`
+            : `${process.env.PUBLIC_URL}/assets/js-inactive.png`
+        }
+        alt="Javascript"
+        width="60px"
+        style={{ backgroundColor: lang === 'js' ? 'white' : 'initial' }}
```

Comment:
@Vipu97 we aren't using inline styles in our project and for the images we are using two different images one for light mode and the other for dark mode no CSS.

Reply:
Ok, I will remove it